### ISSUE TITLE
SC 3.1.5 Reading Level & lower secondary education level

### ISSUE THREAD

**Issue by LaZay** [w3c/wcag3#162]
As it is defined now, the 3.1.5 Reading Level criteria is not an accessibility criteria. It is an audience criteria. A **large audience criteria**. Which is great for large audience publishing. Which is a catastropha for educational publishing (Upper secondary, Higher education), but also scientific, technical, legal, medical, any specialized publishing as a matter of fact. It will **discourage the respect of every other clever AAA criteria**. Because all AAA criteria must be met to reach this WCAG level. Why even try then? :-(

OK to apply this criteria to large audience sites/publishing. It cannot apply to sites/publications that are specifically written for readers with higher education level than the Lower secondary (MCQs, textbooks, etc. contents). Or that are written for experts (specialized sites/books)... These cases should be exceptions to the criteria. Otherwise NO upper education, scientific, technical, legal, medical, any specialized publishing will never offer any AAA service. I am sure nobody wants this.

Some impaired people go to university, other become great experts. They need the highest level of content accessibility to succeed in their studies/career. This includes all AAA criteria. Except for the 3.1.5 one, as it is defined now.

**Comment by cstrobbe** [comment]
Perhaps it is worth mentioning that since the comment was originally posted, ISO has published [ISO 24495-1:2023 Plain language: Part 1: Governing principles and guidelines](https://www.iso.org/standard/78907.html). I'm not saying "lower secondary education level" is the same level as ISO's plain language; the more important point is that the standard describes practices that would be worth mentioning in Understanding 3.1.5.
I would also reword the part about readability formulas to avoid the impression that these can be relied upon to evaluate readability. ("The readability of content may also be determined by applying a readability formula to the selected passage." Not really, as I have learnt.)
